asreview/simulation/simulate.py

```diff
@@ -157,13 +158,29 @@ def review(self):
             record_ids = self.query(self.n_instances)
             labeled = self.label(record_ids)
 
-            pbar_rel.update(labeled["label"].sum())
+            pbar_rel_update = labeled["label"].sum()
+
+            pbar_rel.update(pbar_rel_update)
             pbar_total.update(1)
 
         else:
             pbar_rel.close()
             pbar_total.close()
 
+            padded_results = list(self._results["label"]) + [0] * (
+                len(self.labels) - len(self._results["label"])
+            )
+
+            metrics_block = f"""
+\033[95m----------------\033[0m Metrics Summary \033[95m----------------\033[0m
+Relevant records            : {sum(self._results['label'])}/{sum(self.labels)}
+Records Labeled             : {len(self._results['label'])}/{len(self.labels)}
+Loss                        : {round(metrics.loss(padded_results), 2)}
+WSS@95%                     : {round(metrics.wss(padded_results, 0.95), 2)}
+\033[95m-------------------------------------------------\033[0m
+            """
+            print(metrics_block)
+
```

Comment:
Thanks, let's go for a simple version for now. Output the loss after the review is finished in a single line (with newline before).